src/services/apes/apes.ts

```diff
@@ -37,12 +37,17 @@ export const apes = createApi({
         }),
       }
     ),
+    endowBalance: builder.query<EndowmentBalances, number>({
+      providesTags: ["balance"],
```

Comment:
Just wanted to double-check whether there are any actions in the webapp that would require refetching this balance that we might have forgotten (by calling `invalidateApesTags`)?

Reply:
just removed this tag as there's no point to revalidating this endpoint (takes time for update to reflect) https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2607/commits/0b6c07eda533ac5f3c259dc2813748fe248beab4